lib/cli/github_settings/do_check.js

```diff
@@ -123,175 +135,200 @@ function do_check(subcmd, opts, args, cb) {
     );
 }
 
+function checkBranchProtection(branchName, ctx, next) {
+    assert.string(branchName, 'branchName');
+    assert.object(ctx, 'ctx');
+    assert.func(next, 'next');
+    assert.object(ctx.log, 'ctx.log');
+    assert.object(ctx.octokit, 'ctx.octokit');
+    assert.object(ctx.repo, 'ctx.repo');
+    assert.array(ctx.checkFailures, 'ctx.checkFailures');
```

Comment:
```suggestion
    assert.arrayOfObject(ctx.checkFailures, 'ctx.checkFailures');
```

Reply:
Yep, I'll fix that. I had been worried that maybe arrayOfObject would complain about empty arrays, but no, it just iterates over each item checking that it's an object.